packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -591,6 +589,15 @@ exports.createNodesForContentType = ({
         // The content of an entry is guaranteed to be updated if and only if the .sys.updatedAt field changed
         entryNode.internal.contentDigest = entryItem.sys.updatedAt
 
+        // Link tags
+        if (entryItem?.metadata?.tags) {
```

Comment:
Entryitem is not null. Is metadata ever null?
```suggestion
        if (entryItem.metadata?.tags) {
```